src/Microsoft.TestPlatform.Common/Filtering/FilterHelpers.cs

```diff
@@ -0,0 +1,115 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.VisualStudio.TestPlatform.Common.Filtering
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Globalization;
+    using System.Linq;
+    using System.Text;
+    using Microsoft.VisualStudio.TestPlatform.Common.Resources;
+
+    public static class FilterHelpers
```

Comment:
Can this be internal? LUT has dependency on `Microsoft.TestPlatform.Common`?

Reply:
Yes, made this public for LUT. LUT has dependency on TranslationLayer, which pulls in Common as well.

Reply:
I guess the tokenizer code should remain private to FilterExpression/Condition in this case, don't see a point of exposing them.